### PR TITLE
add the minimum check period attribute on metrics post message

### PR DESCRIPTION
**Pull request by rphillips**

### CODE REVIEW

File: schedule/scheduler.lua [hunk 1]
```diff
@@ -33,6 +35,7 @@ function Scheduler:initialize(checks)
   self._checkMap = {}
   self._checks = {}
   self._runCount = 0
+  self._minCheckPeriod = 0xffff
```
Comment (jirwin) [inline schedule/scheduler.lua]: if the value stays as `0xffff` what will it actually look like in the payload? maybe it is best to start this out at the max check period(30 minutes)?

Comment (rphillips) [inline schedule/scheduler.lua]: fixed in 5e0cd9f

File: check/base.lua [hunk 2]
```diff
@@ -606,10 +606,19 @@ function CheckResult:initialize(check, options)
   self._state = DEFAULT_STATE
   self._status = DEFAULT_STATUS
   self._check = check
+  self._minimumCheckPeriod = 0xffff
```
Comment (jirwin) [inline check/base.lua]: Should probably change the default here as well

**Comment by jirwin** [comment]
+1 Awesome!

**Pull request merged**